core/data/nlu.en.md

```diff
@@ -1,28 +1,19 @@
-## intent:greet
-- hey
-- hello
-- hi
-- hello there
-- good morning
-- good evening
-- moin
-- hey there
-- let's go
-- hey dude
-- goodmorning
-- goodevening
-- good afternoon
+## intent: cancel
+- I changed my mind
+- don't want to enroll
+- cancel
+- I don't want to give my number
 
-## intent:goodbye
-- good afternoon
-- cu
-- good by
-- cee you later
-- good night
-- bye
-- goodbye
-- have a nice day
-- see you around
-- bye bye
-- see you later
+## intent: no_phone
+- I don't have a mobile phone
+- I don't have one
+- I don't own a cellphone
+- I have none
```

Comment:
This might be ok in the short term, but some sentences are either too specific (I don't want to give my number for the generic cancel intent) or too generic (I don't have one) for the no_phone intent.